### PR TITLE
Removes handheld subtype from flashes

### LINKED ISSUE #54538: hypnotic flashes aren't subtypes of handheld flashes
This means that you can't, say, use them to fill in the flash slots of a cyborg head, which is very sad.

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

fixes #54538

## Changelog
:cl:
fix: hypnoflashes are now a subtype of handheld, which lets you do a few more things with them that normal flashes can.
/:cl:

### CODE REVIEW

**Comment by SuperNovaa41** [comment]
Borg hypnoflash eyes time

**Comment by XDTM** [comment]
You didn't repath any of the functions under the hypnoflash though

**Comment by Mothblocks** [comment]
> You didn't repath any of the functions under the hypnoflash though

this

also, if i remember right, the /handheld path shouldn't exist at all, its parent path is the only thing on it. this is a bad fix

File: code/modules/instruments/songs/_song.dm [hunk 1]
```diff
@@ -378,12 +378,12 @@
 				set_linear_falloff_duration(var_value)
 
 // subtype for handheld instruments, like violin
-/datum/song/handheld
+/datum/song
```
Comment (Mothblocks) [inline code/modules/instruments/songs/_song.dm:381]: uhhhhh

File: code/modules/instruments/items.dm [hunk 2]
```diff
@@ -8,7 +8,7 @@
 	lefthand_file = 'icons/mob/inhands/equipment/instruments_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/instruments_righthand.dmi'
 	/// Our song datum.
-	var/datum/song/handheld/song
+	var/datum/song/song
```
Comment (Mothblocks) [inline code/modules/instruments/items.dm:11]: uhhhhhhhhhhhhhhhhhhh

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Add it to https://github.com/tgstation/tgstation/tree/master/tools/UpdatePaths as well

File: code/modules/instruments/items.dm (hunk 2 above)
Comment (Mothblocks) [reply code/modules/instruments/items.dm:11]: yeah tihs is still here @tralezab

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by Mothblocks** [comment]
why did you undraft it has an outstanding review and is conflicting

**Comment by tralezab** [comment]
HOW IS IT STILL THERE

**Comment by tralezab** [comment]
pr is CURSED

https://discord.com/channels/326822144233439242/326831214667235328/838533132038701096

**Pull request closed without merging**